src/utils/constants.ts

```diff
@@ -1 +1,367 @@
+  { value: 'PNG', label: 'Papua New Guinea' },
+  { value: 'PRY', label: 'Paraguay' },
+  { value: 'PER', label: 'Peru' },
+  { value: 'PHL', label: 'Philippines' },
+  { value: 'PCN', label: 'Pitcairn' }, // not found?
+  { value: 'POL', label: 'Poland' },
+  { value: 'PRT', label: 'Portugal' },
+  { value: 'QAT', label: 'Qatar' },
+  { value: 'REU', label: 'Reunion' },
+  { value: 'ROU', label: 'Romania' },
+  { value: 'RUS', label: 'Russia' },
+  { value: 'RWA', label: 'Rwanda' },
+  { value: 'SHN', label: 'Saint Helena' }, // not found?
+  { value: 'KNA', label: 'Saint Kitts and Nevis' },
+  { value: 'LCA', label: 'Saint Lucia' },
+  { value: 'SPM', label: 'Saint Pierre and Miquelon' },
+  { value: 'VCT', label: 'Saint Vincent and the Grenadines' },
+  { value: 'SMR', label: 'San Marino' },
+  { value: 'STP', label: 'Sao Tome and Principe' },
+  { value: 'SAU', label: 'Saudi Arabia' },
+  { value: 'SEN', label: 'Senegal' },
+  { value: 'SCG', label: 'Serbia' },
+  { value: 'SYC', label: 'Seychelles' },
+  { value: 'SLE', label: 'Sierra Leone' },
+  { value: 'SGP', label: 'Singapore' },
+  { value: 'SVK', label: 'Slovakia' },
+  { value: 'SVN', label: 'Slovenia' },
+  { value: 'SLB', label: 'Solomon Islands' },
+  { value: 'SOM', label: 'Somalia' },
+  { value: 'ZAF', label: 'South Africa' },
+  { value: 'SGS', label: 'South Georgia and the South Sandwich Islands' }, // not found?
+  { value: 'ESP', label: 'Spain' },
+  { value: 'LKA', label: 'Sri Lanka' },
+  { value: 'SDN', label: 'Sudan' },
+  { value: 'SUR', label: 'Suriname' },
+  { value: 'SWZ', label: 'Swaziland' },
+  { value: 'SWE', label: 'Sweden' },
+  { value: 'CHE', label: 'Switzerland' },
+  { value: 'SYR', label: 'Syrian Arab Republic' },
+  { value: 'TWN', label: 'Taiwan' },
+  { value: 'TJK', label: 'Tajikistan' },
+  { value: 'TZA', label: 'Tanzania' },
+  { value: 'THA', label: 'Thailand' },
+  { value: 'TLS', label: 'Timor-Leste' },
+  { value: 'TGO', label: 'Togo' },
+  { value: 'TKL', label: 'Tokelau' },
+  { value: 'TON', label: 'Tonga' },
+  { value: 'TTO', label: 'Trinidad and Tobago' },
+  { value: 'TUN', label: 'Tunisia' },
+  { value: 'TUR', label: 'Turkey' },
+  { value: 'TKM', label: 'Turkmenistan' },
+  { value: 'TCA', label: 'Turks and Caicos Islands' },
+  { value: 'TUV', label: 'Tuvalu' },
+  { value: 'UGA', label: 'Uganda' },
+  { value: 'UKR', label: 'Ukraine' },
+  { value: 'ARE', label: 'United Arab Emirates' },
+  { value: 'GBR', label: 'United Kingdom' },
+  { value: 'URY', label: 'Uruguay' },
+  { value: 'UZB', label: 'Uzbekistan' },
+  { value: 'VUT', label: 'Vanuatu' },
+  { value: 'VAT', label: 'Vatican' },
+  { value: 'VEN', label: 'Venezuela' },
+  { value: 'VNM', label: 'Vietnam' },
+  { value: 'VGB', label: 'British Virgin Islands' },
+  { value: 'WLF', label: 'Wallis and Futuna' },
+  { value: 'ESH', label: 'Western Sahara' },
+  { value: 'YEM', label: 'Yemen' },
+  { value: 'ZMB', label: 'Zambia' },
+  { value: 'ZWE', label: 'Zimbabwe' },
```

Comment:
This country list is interesting for sure lol. We can probably leave the lines with comments in, but can we add South Sudan? `{value: 'SSD', label: 'South Sudan'}`